contrib/kind-common

```diff
@@ -348,6 +348,10 @@ install_kubevirt() {
             kubectl logs --all-containers=true -n kubevirt $p || true
         done
     fi
+
+    kubectl -n kubevirt patch kubevirt kubevirt --type=json --patch '[{"op":"add","path":"/spec/configuration/developerConfiguration","value":{"featureGates":[]}},{"op":"add","path":"/spec/configuration/developerConfiguration/featureGates/-","value":"NetworkBindingPlugins"}]'
+    local passt_binding_image="quay.io/kubevirt/network-passt-binding:${kubevirt_version}"
```

Comment:
nit: worth to add please a new line above this line (since other changes are likely to be done anyhow https://github.com/ovn-org/ovn-kubernetes/pull/4655/files#r1741485104

Reply:
DONE